crm_compassion/models/event_compassion.py

```diff
@@ -108,7 +108,7 @@ def _compute_analytic_lines(self):
             if event.analytic_id:
                 expenses = analytic_line_obj.search([
                     ('account_id', '=', event.analytic_id.id),
-                    ('amount', '<', '0.0')])
+                    ('amount', '>', '0.0')])

```

Comment:
After better checking, this is false. Previous was correct. You can remove this change, sorry for the confusion. I will wait for @davidwul  to explain how we can add new expenses.